webextensions/api/privacy.json

```diff
@@ -81,7 +81,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "≤61"
+                  "version_added": "18"
```

Comment:
✅ https://github.com/chromium/chromium/commit/607e6522f841933e0d3130c09d33207876850948 landed in [18.0.1018.0](https://github.com/chromium/chromium/releases/tag/18.0.1018.0) before 18.0.1025.